pyqtorch/time_dependent/methods/krylov.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import torch
+from torch import Tensor
+
+from pyqtorch.time_dependent.integrators.krylov import KrylovIntegrator
+
+
+class Krylov(KrylovIntegrator):
```

Comment:
Does it deserve a base class since it is used only once ? Or is it possible to abstract `Integrator` even more to be use by all solvers ?

Reply:
Krylov integrator is a bit different than the adaptive one so I doubt that we'll be able to abstract that. However, I believe that we can come up with master equation solver that uses Krylov method. Thus, I think this class has to stay.